### DISCUSSION TITLE
Change ID field type to Integer

### DISCUSSION THREAD

**Discussion by nyudenkov** [fastapi-users/fastapi-users#503] [Q&A] [answered]
Hello there! How can I painlessly change user.id field to int type?

**Comment by nyudenkov** [comment]
@frankie567

**Comment by frankie567** [comment, marked as answer]
Unfortunately, this library is designed to work with UUIDs (models, routes...). I fear there is no easy way to change it out-of-the-box. But feel free to fork it and adapt it to your needs 👍

**Reply by nyudenkov** [reply]
> Unfortunately, this library is designed to work with UUIDs (models, routes...). I fear there is no easy way to change it out-of-the-box. But feel free to fork it and adapt it to your needs 👍

Thank you for answer! 🤘
